netjsonconfig/backends/base/backend.py

```diff
@@ -380,7 +380,7 @@ def _generate_contents(self, tar):
         # create a file for each VPN
         for vpn in vpn_instances:
             lines = vpn.split('\n')
-            vpn_name = lines[0]
+            vpn_name = lines[0].split(self.config_suffix)[0]
```

Comment:
I have made this change because while rendering the [zerotier configuration, I included `(.json)` in the comment](https://github.com/openwisp/netjsonconfig/pull/272/files#diff-1ad7f38c2f702da652381ecfb5a3ddae744d893f0f84cf61ca6a7ba2e7c71bc8R2)). This resulted in generating a zerotier configuration `tarfile` with the `.json.json` extension, as shown below:

```bash
(env) ➜  zerotier git:(issue-208/add-zerotier-backend) ✗ py -m unittest test_backend.TestBackend
..[<TarInfo '9536600adf654321.json.json' at 0x7fa2dcb3ff40>, <TarInfo '9536600adf654322.json.json' at 0x7fa2dcb3fe80>]

therefore to avoid this I have spilt `lines[0]` with `config_suffix`.